README.rst

```diff
@@ -16,7 +16,47 @@ The mixins support optional checksums of arbitrary columns, and asynchronous pro
 Documentation
 -------------
 
-The full documentation is at https://super-csv.readthedocs.org.
+Super CSV is a stand alone library that can be used for CSV management, both syncronous and async.
+
+Testing
+-------
+::
+
+    $make requirements
+
+Will install all pre-requisuites. ::
+
+    $make test
+
+Runs the unit tests in local environment.
+
+Using with Docker Devstack
+--------------------------
+Prerequisite: Have your Open edX https://github.com/edx/devstack properly installed.
+Note: When you see "from inside the lms" below, it means that you've run ``make lms-shell`` from your devstack directory
+and are on a command prompt inside the LMS container.
+
+#. Clone this repo into ../src/ directory (relative to your "devstack" repo location). This will mount the directory in
+a way that is accessible to the lms container.
+
+#. From inside the lms, uninstall super-csv and reinstall your local copy. You can just copy the following line. This is
+necessary if one wants to use latest version for testing/development purposes::
+
+    pip uninstall super-csv -y; pip install -e /edx/src/super-csv
+
+
+#. Setup dev environment (since virtual environments are recommended for python development, here is an example of using
+virtualenv. Other tools would work as well). By default, edx containers come with virtualenv preinstalled ::
+
+    cd /edx/src/super-csv
+    virualenv super-csv-env
```

Comment:
i think you're still missing a 't'

Reply:
Thank you! First initial of my last name :)